gen/calc.py

```diff
@@ -579,6 +579,12 @@ def fz_err(msg):
             validate_int_in_range(port, 1, 65535)
 
 
+def calculate_fair_sharing_excluded_resource_names(gpus_are_scarce):
```

Comment:
I actually thought about using this name, but none of the other functions follow this conventions, so I decided to leave it out.